app/src/main/res/values/strings.xml

```diff
@@ -16,6 +16,10 @@
         <item>@string/edit</item>
     </array>
     <string name="options_for">Options for:&#160;</string>
+    <string name="make_groups">Make Groups</string>
+    <string name="no_groups_created_message">
+        Use the {ion-android-settings} icon to customize your groups
+        and then tap \"Make Groups\" at the bottom</string>
```

Comment:
nit: Small formatting change so developers can focus on the string and adding a period since this is a complete sentence.

```suggestion
        and then tap \"Make Groups\" at the bottom.
    </string>
```